src/placeos-models/asset.cr

```diff
@@ -10,13 +10,10 @@ module PlaceOS::Model
     attribute category : String = ""
     attribute description : String = ""
 
-    # TODO: `size` needs further definition
-    attribute size : String = ""
-
```

Comment:
Not sure we entirely dropped it - but seems like a good idea if we're not really doing anything with it. @MrYuion I think we're dropping the size field here.

@tassja could you please update the specification if possible